aten/src/ATen/ConjugateFallback.cpp

```diff
@@ -0,0 +1,42 @@
+#include <ATen/ConjugateFallback.h>
+#include <ATen/native/UnaryOps.h>
+
+namespace at {
+
+    TORCH_LIBRARY_IMPL(_, Conjugate, m) {
+        m.fallback(torch::CppFunction::makeFromBoxedFunction<&conjugateFallback>());
+    }
+
+   Tensor conj_materialize(const Tensor& self) {
+        // this function assumes that the tensor input has it's conjugate bit set
+        self.set_conjugate(false);
+        Tensor self_conjugated = self.conj();
+        return self_conjugated;
+    }
+
+    void conjugateFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+        // Unwrap all arguments
+        const auto num_arguments = op.schema().arguments().size();
+        auto arguments = torch::jit::pop(*stack, num_arguments);
+
+        // conjugate each tensor argument on the stack with it's conjugate DispatchKey + bit set
+        // leave all other arguments unchanged
+        for (int64_t idx = 0; idx < arguments.size(); ++idx) {
+            const auto& ivalue = arguments[idx];
+            if (!ivalue.isTensor()) {
+                torch::jit::push(stack, ivalue);
+                continue;
+            }
+            auto* impl = ivalue.unsafeToTensorImpl();
+            if (impl->is_conjugate()) {
+                const auto& tensor = ivalue.toTensor();
+                auto conjugated_tensor = conj_materialize(tensor);
+                torch::jit::push(stack, conjugated_tensor);
+            } else {
+                torch::jit::push(stack, ivalue);
```

Comment:
nit: You can improve the performance of this code by being more careful about moving ivalues on and off the argument stack. This will help you avoid unnecessary reference count bumps. So you would move an argument off the arguments list, and the move it back onto the stack when you push it.

Reply:
I'm going to try to fix this: instead of popping all arguments off the stack and pushing them all back on (most of which are unchanged except for the conjugated tensor), I'm just going to selectively overwrite the memory in the stack (vector) for the tensors that need to be conjugated. Let me know if that's what you had in mind.

Reply:
That will be even better :)